tests/test-extensions/heisenberg-extension/src/main/java/org/mule/test/heisenberg/extension/HeisenbergSource.java

```diff
@@ -135,7 +135,7 @@
   @NullSafe
   private Map<String, Object> debtProperties;
 
-  @Parameter
+  @org.mule.sdk.api.annotation.param.Parameter
```

Comment:
add a comment explaining that you're intentionally mixing annotations in order to test all code flows